examples/fluids/navierstokes.c

```diff
@@ -765,6 +765,10 @@ int main(int argc, char **argv) {
   ierr = PetscOptionsScalar("-CtauS",
                             "Scale coefficient for tau (nondimensional)",
                             NULL, CtauS, &CtauS, NULL); CHKERRQ(ierr);
+  if (stab == STAB_NONE && CtauS != 0) {
+    ierr = PetscPrintf(comm, "Warning! Use -CtauS only with -stab su or -stab supg\n");
+    CHKERRQ(ierr);
+  }
```

Comment:
@valeriabarra , thank you for adding this warning. Actually, `CtauS` and `strong_form` are not used in `density densitycurrent`. Maybe adding warning for those too?!

Reply:
I think we don't need line 64 and 65 of `densitycurrent.h`.

Reply:
Thanks Leila!